src/lightning/app/cli/commands/cp.py

```diff
@@ -162,6 +174,45 @@ def _upload(source_file: str, presigned_url: ApplyResult, progress: Progress, ta
     file_uploader.upload()
 
 
+def _zip_files(live: Live, remote_src: str, local_dst: str) -> None:
+    live.stop()
```

Comment:
Let's move the live right before the progress.start. Everything below as a small delay.